plugins/forc-index/src/commands/build.rs

```diff
@@ -18,11 +18,12 @@ pub struct Command {
     #[clap(long, default_value = defaults::INDEXER_TARGET, help = "Target at which to compile.")]
     pub target: String,
 
-    /// Build optimized artifacts with the release profile.
+    /// Build optimized artifacts with the release profile. Set to true by default.
```

Comment:
We don't have to mention that it's set to true by default. Clap will automagically show this when the user runs `--help`